App/Sagas/Account/AccountSagas.ts

```diff
@@ -47,7 +48,7 @@ export function * refreshPeerId() {
   while (true) {
     try {
       yield take(getType(AccountActions.refreshPeerIdRequest))
-      const peerIdResult = yield call(API.ipfs.peerId)
+      const peerIdResult = yield call(Textile.ipfs.peerId)
```

Comment:
No more `API` export from the sdk. The whole API just available from the default export (called `Textile` here). This matches the ios and android sdk structure exactly.